tests/test_nest_asyncio.py

```
@@ -12,60 +12,80 @@
+config = RailsConfig.from_content(yaml_content="models: []")
+chat = TestChat(config, llm_completions=["Hello there!"])
+"""
 
 
-def test_sync_api():
-    chat >> "Hi!"
-    chat << "Hello there!"
+def _run_in_subprocess(source, disable_nest_asyncio):
+    env = os.environ.copy()
+    env["DISABLE_NEST_ASYNCIO"] = str(disable_nest_asyncio)
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)